Makefile

```diff
@@ -27,15 +27,18 @@ rounduptest:
 flake8:
 	flake8 scripts/
 
+python_unittest:
+	python -m unittest discover -v -s scripts/lib/libvirt/
+
 # for travis-CI:
 install: debianinstall genericinstall
 
 debianinstall:
 	sudo apt-get update -qq
-	sudo apt-get -y install libxml-libxml-perl libjson-xs-perl
+	sudo apt-get -y install libxml-libxml-perl libjson-xs-perl python-libvirt
 
 suseinstall:
-	sudo zypper install perl-JSON-XS perl-libxml-perl
+	sudo zypper install perl-JSON-XS perl-libxml-perl libvirt-python
 
 genericinstall:
 	sudo pip install bashate flake8
```

Comment:
You may also install libvirt-python in the generic install via. pip

$ pip install libvirt-python